build.gradle

```diff
@@ -21,11 +21,11 @@ repositories {
 dependencies {
 	implementation 'org.springframework.boot:spring-boot-starter-jdbc'
 	implementation 'org.springframework.boot:spring-boot-starter-web'
+	implementation 'org.springframework.boot:spring-boot-starter-validation'
```

Comment:
See [this guide](https://www.baeldung.com/spring-boot-bean-validation).  Enables validation of params to rest controllers within Spring Boot